pages/linux/ltrace.md

```diff
@@ -0,0 +1,19 @@
+# ltrace
+
+> Display dynamic library calls of a process
+
+- Print (trace) library calls of a binary file `{{program}}`.
+
+`ltrace {{./program}}`
```

Comment:
The ./ isn't part of the placeholder, since it will be present for any program you'd want to run this way.